Formula/ponyc.rb

```diff
@@ -13,17 +12,27 @@ class Ponyc < Formula
     sha256 "007c43a75741ecd4c7c96c9fc67d816ebc845c0aac94687c4e2cbb1ee959ebfb" => :mavericks
   end
 
-  depends_on "llvm" => "with-rtti"
+  depends_on "llvm"
   depends_on "libressl"
   depends_on "pcre2"
   needs :cxx11
 
   def install
     ENV.cxx11
-    system "make", "install", "config=release", "destdir=#{prefix}", "verbose=1"
+    ENV["LLVM_CONFIG"]="#{Formula["llvm"].bin}/llvm-config"
```

Comment:
No, that's unrelated to the `keg_only` status. `.bin` just expands to `Cellar/<formula>/<version>/bin` while `.opt_bin` expands to `opt/<formula>/bin` (where `opt/<formula>` is a symlink created by `brew install` et al.). We try to avoid using the versioned paths when referencing formulae from other formulae, in case those formulae hard-code the paths somewhere in their binaries, configuration files, etc. Which would be a problem once the dependency is updated (and the path thus changes). In this particular case it doesn't make much of a difference because `ponyc` statically links to the `llvm` library (or libraries).